src/fiatExchanges/quotes/FiatConnectQuote.ts

```diff
@@ -340,4 +346,30 @@ export default class FiatConnectQuote extends NormalizedQuote {
   getTokenId(): string {
     return this.tokenId
   }
+
+  async getTransferResult(flow: CICOFlow, fiatAccountId: string): Promise<TransferOutResponse> {
```

Comment:
this is essentially the same function as [the previous saga](https://github.com/valora-inc/wallet/blob/00ffa39770126ec3fa7dd5fa5d16b82b10cd0c77/src/fiatconnect/saga.ts#L910-L943), but adding it as a class method so that it can be used in a react hook